lxd/main_cluster.go

```diff
@@ -410,34 +440,9 @@ func (c *cmdClusterRecoverFromQuorumLoss) Run(cmd *cobra.Command, args []string)
 	return cluster.Recover(db)
 }
 
-func (c *cmdClusterRecoverFromQuorumLoss) promptConfirmation() error {
-	reader := bufio.NewReader(os.Stdin)
-	fmt.Print(`You should run this command only if you are *absolutely* certain that this is
-the only database node left in your cluster AND that other database nodes will
-never come back (i.e. their LXD daemon won't ever be started again).
-
-This will make this LXD instance the only member of the cluster, and it won't
-be possible to perform operations on former cluster members anymore.
-
-However all information about former cluster members will be preserved in the
-database, so you can possibly inspect it for further recovery.
-
-You'll be able to permanently delete from the database all information about
-former cluster members by running "lxc cluster remove <member-name> --force".
-
-See https://documentation.ubuntu.com/lxd/en/latest/howto/cluster_recover/#recover-from-quorum-loss for more
-info.
-
-Do you want to proceed? (yes/no): `)
-	input, _ := reader.ReadString('\n')
-	input = strings.TrimSuffix(input, "\n")
-
-	if !shared.ValueInSlice(strings.ToLower(input), []string{"yes"}) {
-		return fmt.Errorf("Recover operation aborted")
-	}
-
-	return nil
-}
+const removeRaftNodePrompt = `You should run this command only if you ended up in an
+inconsistent state where a node has been uncleanly removed (i.e. it doesn't show
```

Comment:
lets use "member" rather than "node"

Reply:
oh, but is this specifically about raft nodes and not LXD members? If so then this is fine.

Reply:
Perhaps use "raft node" rather than just "node" though to be clear.

Reply:
I think "cluster member" is meant here, as "raft node" is probably not very meaningful to a user, even if they're very familiar with LXD clustering.